### DISCUSSION TITLE
How to monitor networks with hidden SSID's

### DISCUSSION THREAD

**Discussion by CoMcE** [nzymedefense/nzyme#617] [Questions]
Hello, 

I installed nzyme and wanted to add my Networks now, but my SSID's are hidden.
Is it possible to monitor them or do I have to make them visible?
If I could leave them hidden, where can I find the fingerprints to add them to nzyme?

Thanx

**Comment by lennartkoopmann** [comment]
Nzyme does currently not support hidden SSIDs.
